cli/src/pcluster/cluster_model.py

```diff
@@ -115,6 +116,53 @@ def _ec2_run_instance(self, pcluster_config, **kwargs):  # noqa: C901 FIXME!!!
                     "Please double check your cluster configuration.\n{1}".format(kwargs["InstanceType"], message)
                 )
 
+    def _get_official_image_id(self, os, architecture):
+        """Return the id of the current official image, for the provided os-architecture combination."""
+        try:
+            images = (
+                boto3.client("ec2")
+                .describe_images(
+                    Filters=[
+                        {
+                            "Name": "name",
+                            "Values": ["{0}*".format(self._get_official_image_name_prefix(os, architecture))],
+                        },
+                        {"Name": "owner-alias", "Values": ["amazon"]},
+                        {"Name": "is-public", "Values": ["true"]},
```

Comment:
this can be problematic between the time we bump version and the time we release the AMIs. probably for now we just want to remove this and rely on the canaries to make sure the AMIs are public after we release.

Reply:
Done